src/otserv.cpp

```diff
@@ -183,6 +197,8 @@ int main(int argc, char* argv[]) {
 #else
 	spdlog::set_pattern("[%Y-%d-%m %H:%M:%S.%e] [%^%l%$] %v ");
 #endif
+	// Disable the force close button
+	closeButton();
```

Comment:
I think this should be configurable. Maybe add a config luankey that allow you to turn this on/off?